### PR TITLE
Introduce an animated header

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 
As always, see [http://strelyaev-w8:9090/testcafe/documentation/getting-started/](http://strelyaev-w8:9090/testcafe/documentation/getting-started/)

### CODE REVIEW

File: src/_includes/head.html [hunk 1]
```diff
@@ -13,4 +13,21 @@
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}">
   <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.0.0/highlight.min.js"></script>
   <script>hljs.initHighlightingOnLoad();</script>
+  <script>
+      window.addEventListener('scroll', function(e) {
+          var body = document.getElementsByTagName('body')[0];
+          var siteHeader = body.querySelector('.site-header');
+          if (!siteHeader) return;
```
Comment (kirovboris) [inline src/_includes/head.html]: Why does Formatting not correspond our code rules? It should be:
```javascript 
var body       = document.getElementsByTagName('body')[0];
var siteHeader = body.querySelector('.site-header');

if (!siteHeader) 
    return;    

if (window.pageYOffset === 0 || body.clientWidth <= 980)
    siteHeader.className = siteHeader.className.replace(/\sminimized\b/,'');
else if(siteHeader.className.indexOf('minimized') === -1)
    siteHeader.className += ' minimized';
```

**Comment by MargaritaLoseva** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Pull request merged**